### ISSUE TITLE
Figure out a way to display sibling articles for a category

### ISSUE THREAD

**Issue by facuspagnuolo** [aragon/help#24]
The first time I hit the "user guide" button in the Aragon Court dashboard it took me to (https://help.aragon.org/article/41-aragon-court) which is actually an article that looks like this:

![image](https://user-images.githubusercontent.com/6967192/77069096-9015a480-69c6-11ea-8365-1b1de5af1e1e.png)

Initially, I didn't know it was an article and that there was more content about Aragon Court. It is not obvious that clicking "Aragon Court" in the sidebar was going to show me other articles about it.

Maybe we can see if there is a way to list all the articles for a category on the sidebar for example

**Comment by sohkai** [comment]
May be a duplicate of https://github.com/aragon/help/issues/2.

**Comment by facuspagnuolo** [comment]
Sounds like it is, I'll let @john-light decide whether we should close it or not

**Comment by facuspagnuolo** [comment]
Pinging @Evalir to see if you can give us a hand with this :)

**Comment by Evalir** [comment]
Will look into this after we get the court-backend deployment unblocked @facuspagnuolo !

**Comment by delfipolito** [comment]
As a quick fix, I put at the bottom of each article  a section called “related articles” with all the articles that belongs to the same category that the one the user is reading. So now it should be more intuitive that there is more content on that topic.

**Comment by john-light** [comment]
Going to close this for now, thanks @delfipolito. If we think of a better solution later we can come back to it.
